pipelines/LSSTCam/nightly-validation.yaml

```diff
@@ -0,0 +1,65 @@
+description: >
+  LSSTCam specialization of the Nightly Validation Pipeline.
+  Run overnight: step1a-single-visit-detectors, step1b-single-visit-visits
+  Run in morning: step1c-single-visit-tracts,step1d-single-visit-global,stage3-coadd
+
+  This is the cheapest possible version of a Nightly Validation pipeline.
+  It excludes PIFF and Object Table generation.
+instrument: lsst.obs.lsst.LsstCam
+imports:
+  - location: $DRP_PIPE_DIR/pipelines/_ingredients/base-v2.yaml
+    labeledSubsetModifyMode: EDIT
+    exclude:
+      # unlike DRP.yaml, do run all analysis-visit-initial tasks, but remove:
+      - analysis-visit-recalibrated
+      - analysis-visit-source
+      # Remove entire recalibration stage including PIFF
+      - stage2-recalibrate
+      - stage4-measure-variability
+      # Remove template generation
+      - selectTemplateCoaddVisits
+      - assembleTemplateCoadd
+      # Remove object table generation
+      - mergeObjectDetection
+      - deblendCoaddFootprints
+      - measureObjectUnforced
+      - mergeObjectMeasurement
+      - measureObjectForced
+      - fitDeepCoaddPsfGaussians
+      - fitDeblendedObjectsSersic
+      - rewriteObject
+      - standardizeObject
+      - consolidateObject
+      - splitPrimaryObject
+      - analyzeObjectTableCore
+      - catalogMatchTract
+      - refCatObjectTract
+      - photometricCatalogMatch
+      - photometricRefCatObjectTract
+      - plotPropertyMapTract
+      - validateObjectTableCore
+      - computeObjectEpochs
+      - analyzeObjectTableSurveyCore
+      - makeMetricTableObjectTableCore
+      - objectTableCoreWholeSkyPlot
+      - makeMetricTableObjectTableCoreRefCatMatch
+      - objectTableCoreRefCatMatchWholeSkyPlot
```

Comment:
It might be a nice pattern to declare some subsets in `base-v2.yaml` (e.g. `make-object-table`, `dia-all`) and exclude those.  Even if they aren't actually used in other places, it keeps the lists of related tasks that might need to be modified in concert together.

Reply:
That is a good idea (esp in the future when we can make subsets of subsets) but given that running `analyze-initial-visits` on its own was raising, I wasn't going to start making a subset with overlapping tasks on this ticket.